pfcpiface/messages.go

```diff
@@ -521,7 +570,7 @@ func (pc *PFCPConn) handleSessionDeletionRequest(upf *upf, msg message.Message,
 		return sendError(fmt.Errorf("Session not found: %v", localSEID))
 	}
 
-	cause := upf.sendMsgToUPF("del", session.pdrs, session.fars)
+	cause := upf.sendMsgToUPF("del", session.pdrs, session.fars, nil)
```

Comment:
comment for why we don't need not delete QER

Reply:
currently we don't write qer to the fast path tables in p4.. once we integrate with bess we can send qer also in delete..